src/service/middlewares/articleExist.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const {HttpCode} = require(`../../constants`);
+
+module.exports = (service) => (req, res, next) => {
+  const {articleId} = req.params;
+  const article = service.findOne(articleId);
+
+  if (!article) {
+    return res.status(HttpCode.NOT_FOUND)
+      .send(`Article with ${articleId} not found`);
+  }
+
+  res.locals.article = article;
```

Comment:
```suggestion
  res.locals.article = article;
```
локаллсы лучше по максимум избегать